content/applications/finance/accounting/receivables/customer_payments/batch.rst

```diff
@@ -41,60 +27,55 @@ Deposit multiple payments in batch
 Record payments to deposit in batch
 -----------------------------------
 
-:doc:`Register the payments <recording>` on the bank account on which you plan to deposit them by
-opening the *Customer Invoice* for which you received a payment, and clicking on *Register Payment*.
-There, select the appropriate Journal linked to your bank account and select *Batch Deposit* as
-Payment Method.
+You can register a received payment by opening the corresponding customer invoice, and clicking on
+:guilabel:`Register Payment`.
+There, select the appropriate :guilabel:`Journal` linked to your bank account and select
+:guilabel:`Batch Deposit` as Payment Method.
 
-.. image:: batch/batch-register-payment.png
+.. image:: batch/batch-payments.png
    :align: center
    :alt: Registering a customer payment as part of a Batch Deposit in Odoo Accounting
 
 Do this step for all checks or payments you want to process in batch.
```

Comment:
I would remove this sentence from here and include it above (as suggested above)